File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 5 - C/q2.c
//Faça um algoritmo que receba N número inteiros e conte quantos são números pares e quantos são
//ímpares. No final, imprima o resultado.

#include<stdio.h>
int main(){
	int n, i, par, impar;
	impar=0;
	par=0;
	for(i=1; i<=5; i++){
		printf("Insira um numero: ");
		scanf("%d", &n);
		if(n%2==0){
			par=par+1;
		}
		else{
			impar=impar+1;
		}
	}
	printf("Par: %d\nImpar: %d\n", par, impar);
}

File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 5 - Python/q2.py
#Faça um algoritmo que receba N número inteiros e conte quantos são números pares e quantos são
#ímpares. No final, imprima o resultado.

par=0
impar=0
for i in range(5):
    x=int(input("Insira o número: "))
    if x%2==0:
        par=par+1
    else:
        impar=impar+1
print("Quantidade de pares: ", par, "\nQuantidade de impares: ", impar)
        
